src/core/reducers/autocomplete.js

```diff
@@ -0,0 +1,50 @@
+export const AUTOCOMPLETE_LOADED = 'core/autocomplete/AUTOCOMPLETE_LOADED';
```

Comment:
With the ducks pattern, I think it's more helpful if they remain in this file. That way you don't have to open a new file to reference the value (since this file will be the only place they are ever referenced).